corehq/apps/cleanup/management/commands/evaluate_couch_model_for_sql.py

```diff
@@ -12,20 +12,14 @@
 
 class Command(BaseCommand):
     """
-        Given a set of attributes and a couch document type, Iterates over all documents
-        and reports back on whether those attributes are ever null and what their largest
-        values are.
+        Given a couch document type, iterates over all documents and reports back
+        on whether those attributes are ever null and what their largest values are.
```

Comment:
Would it make sense to say "longest" instead of "largest"? "Largest" has different meanings in different contexts (e.g., largest integer vs largest string).